Libplanet.Extensions.Cocona/Commands/StoreQueryCommand.cs

```diff
@@ -0,0 +1,142 @@
+            {
+                Console.Error.WriteLine("cannot find the main branch of the blockchain");
+                return;
+            }
+
+            BlockHash? blockHash = store.IndexBlockHash(chainId.Value, blockIndex);
+            if (blockHash == null)
+            {
+                Console.Error.WriteLine($"cannot find the block with the height[{blockIndex}]" +
+                                        $" within the blockchain[{chainId}]");
+                return;
+            }
+
+            Block<DummyAction> block = store.GetBlock<DummyAction>(blockHash.Value);
+            if (block == null)
+            {
+                Console.Error.WriteLine($"cannot find the block with the hash[{blockHash}]");
+            }
+
+            Console.WriteLine(SerializeHumanReadable(block));
+        }
+
+        [Command(Description = "Query block by the hash")]
+        public void BlockByHash(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("BLOCK-HASH", Description = "block hash")]
+            string blockHash
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            var block = store.GetBlock<DummyAction>(BlockHash.FromString(blockHash));
+            if (block == null)
+            {
+                Console.Error.WriteLine($"cannot find the block with the hash[{blockHash}]");
+                return;
+            }
+
+            Console.WriteLine(SerializeHumanReadable(block));
+        }
+
+        [Command(Description = "Query transaction by the tx id")]
+        public void TxById(
+            [Argument("HOME", Description = "root home path")]
+            string home,
+            [Argument("TX-ID", Description = "tx id")]
+            string strTxId
+        )
+        {
+            var store = new RocksDBStore.RocksDBStore(home);
+            TxId txId = TxIdFromString(strTxId);
+            Transaction<DummyAction> tx = store.GetTransaction<DummyAction>(txId);
+            if (tx == null)
+            {
+                Console.Error.WriteLine($"cannot find the tx with the tx id[{strTxId}]");
+                return;
+            }
+
+            Console.WriteLine(SerializeHumanReadable(tx));
+        }
+
+        private static string SerializeHumanReadable<T>(T target) =>
+            JsonSerializer.Serialize(
+                target,
+                new JsonSerializerOptions
+                {
+                    WriteIndented = true,
+                    Converters =
+                    {
+                        new ByteArrayStringJsonConverter(),
+                    },
+                }
+            );
+
+        private static TxId TxIdFromString(string hex) =>
+            new TxId(ByteUtil.ParseHex(hex ?? throw new ArgumentNullException(nameof(hex))));
+
+        private class DummyAction : IAction
+        {
+            public DummyAction()
+            {
+                PlainValue = default(Null);
```

Comment:
```suggestion
                PlainValue = Null.Value;
```

See <https://github.com/planetarium/bencodex.net/pull/37> as well for details.